pkgs/applications/office/zim/default.nix

```diff
@@ -16,12 +25,25 @@ python3Packages.buildPythonApplication rec {
     sha256 = "13vhwsgv6mscgixypc0ixkgj0y7cpcm7z7wn1vmdrwp7kn8m3xgx";
   };
 
-  buildInputs = [ gtk3 gobject-introspection wrapGAppsHook gnome3.adwaita-icon-theme ];
-  propagatedBuildInputs = with python3Packages; [ pyxdg pygobject3 ];
+  buildInputs = [
+    gtk3
+    pango
+    gnome3.adwaita-icon-theme
+    gobject-introspection
+  ];
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+  ];
 
+  # wrapGAppsHook seems to be not cross-compile ready
+  strictDeps = false;
```

Comment:
Maybe someone can explain me what is going on here.

Reply:
See https://github.com/NixOS/nixpkgs/issues/56943